crates/assists/src/handlers/generate_function.rs

```diff
@@ -36,8 +36,8 @@ use crate::{
 //     bar("", baz());
 // }
 //
-// fn bar(arg: &str, baz: Baz) {
-//     ${0:todo!()}
+// fn bar(arg: &str, baz: Baz) $0 {
+//     todo!()
```

Comment:
Shouldn't we make `todo!` the second tab-stop then?

And it probably makes sense to use `-> ` for the first tab stop, which should be easy to remove or complete.

Reply:
I tried to add second tab stop but it seems it's not available for assists. Am I wrong ?

Reply:
Ah, indeed, this is not yet supported on the client, and would require some non-trivial engineering  (IIRC, we can't just leverage existing snippet support, because VS Code auto-indents snippets).